test/test_device_radix_sort.cu

```diff
@@ -1115,7 +1115,7 @@ void TestValueTypes(
     // Initialize the solution
     NumItemsT *h_reference_ranks = NULL;
     KeyT *h_reference_keys = NULL;
-    InitializeSolution<IS_DESCENDING>(h_keys, num_items, num_segments, pre_sorted, h_segment_offsets, begin_bit, end_bit, h_reference_ranks, h_reference_keys);
+    InitializeSolution<IS_DESCENDING, TEST_VALUE_TYPE != 0>(h_keys, num_items, num_segments, pre_sorted, h_segment_offsets, begin_bit, end_bit, h_reference_ranks, h_reference_keys);
```

Comment:
Add a brief comment above this line explaining why we only want ranks for TEST_VALUE_TYPE != 0.